src/tail/tail.rs

```diff
@@ -101,6 +107,36 @@ pub fn uumain(args: Vec<String>) -> i32 {
         };
     }
 
+    match given_options.opt_str("pid") {
+        Some(pid_str) => {
```

Comment:
If the error is ignorable, I would like to use `if let Some(...) = ... {}`. e.g.:

```rust
if let Some(pid_str) = given_options.opt_str("pid") {
    if let Ok(pid) = pid_str.parse() {
    ....
    }
}
```